tox.ini

```diff
@@ -1,38 +1,36 @@
 [tox]
-envlist = flake8,typing,black,py36,py37,py38
+envlist = bandit,flake8,typing,black,py
```

Comment:
I like having these listed here, currently I run just `tox` and it runs the tests all supported Python versions.

Reply:
You have all versions installed? :open_mouth: 

I expect most people to only have 1 Python 3 version installed; for them, `tox` will fail doe to the missing Python versions.